x-pack/plugins/upgrade_assistant/public/application/mount_management_section.tsx

```diff
@@ -1,67 +0,0 @@
-/*
```

Comment:
Ok makes sense! I wasn't sure about leaving it or not because it was just a few lines. I've added it back and I'm also initialising api and breadcrumbs there as suggested